Invalid link for https://www.ietf.org/rfc/rfc4559.txt

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

docs

### Describe the bug

There is a new link error in the documentation:

```
[ERROR] Failures: 
[ERROR]   ExternalLinksTest.checkExternalLinks:41 Broken links (1) in guide 'server_admin': 
		- https://www.ietf.org/rfc/rfc4559.txt (invalid redirect to https://www.rfc-editor.org/rfc/rfc4559.txt)
```

See for example this PR run: https://github.com/keycloak/keycloak/actions/runs/31080033269/job/92546575383

### Version

999.0.0-SNAPSHOT

### Regression

- [ ] The issue is a regression

### Expected behavior

The documentation links test should run OK.

### Actual behavior

The links Keycloak Documentation tests fail.

### How to Reproduce?

``` 
cd docs/documentation
mvn clean install
```

### Anything else?

_No response_

--- a/docs/documentation/server_admin/topics/authentication/kerberos.adoc
+++ b/docs/documentation/server_admin/topics/authentication/kerberos.adoc
@@ -18,7 +18,7 @@ A typical use case for web authentication is the following:
 
 [WARNING]
 ====
-The https://www.ietf.org/rfc/rfc4559.txt[Negotiate] www-authenticate scheme allows NTLM as a fallback to Kerberos and on some web browsers in Windows NTLM is supported by default.  If a www-authenticate challenge comes from a server outside a browsers permitted list, users may encounter an NTLM dialog prompt.  A user would need to click the cancel button on the dialog to continue as {project_name} does not support this mechanism.  This situation can happen if Intranet web browsers are not strictly configured or if {project_name} serves users in both the Intranet and Internet.  A https://github.com/keycloak/keycloak/issues/8989[custom authenticator] can be used to restrict Negotiate challenges to a whitelist of hosts.
+The https://datatracker.ietf.org/doc/html/rfc4559[Negotiate] www-authenticate scheme allows NTLM as a fallback to Kerberos and on some web browsers in Windows NTLM is supported by default.  If a www-authenticate challenge comes from a server outside a browsers permitted list, users may encounter an NTLM dialog prompt.  A user would need to click the cancel button on the dialog to continue as {project_name} does not support this mechanism.  This situation can happen if Intranet web browsers are not strictly configured or if {project_name} serves users in both the Intranet and Internet.  A https://github.com/keycloak/keycloak/issues/8989[custom authenticator] can be used to restrict Negotiate challenges to a whitelist of hosts.
 ====
 
 Perform the following steps to set up Kerberos authentication:

